### DISCUSSION TITLE
Can't import obfuscated spring boot project into another (Component scan failed for imported module in the parent module)

### DISCUSSION THREAD

**Discussion by SIR4GO** [devslm/proguard-spring-boot-example#7] [Q&A]
Hi,  i'm realy appreciate the support,
Can't import obfuscated spring boot project into another (Component scan failed for imported module in the parent module)

**Comment by devslm** [comment]
Hi!
I've never done this so I can't answer your question.

**Comment by devslm** [comment]
Perhaps this is because another project is trying to find the components by the original name.

**Reply by SIR4GO** [reply]
thank you.
